tests/functional/api/groups/members_test.py

```diff
@@ -21,7 +22,16 @@ def test_it_returns_list_of_members_for_restricted_group_without_authn(
         res = app.get("/api/groups/{pubid}/members".format(pubid=group.pubid))
 
         assert res.status_code == 200
-        assert len(res.json) == 3
+        assert res.json == [
+            {
+                "authority": membership.group.authority,
+                "userid": membership.user.userid,
+                "username": membership.user.username,
+                "display_name": membership.user.display_name,
+                "roles": membership.roles,
+            }
+            for membership in group.memberships
+        ]
```

Comment:
Made this test assertion more specific so I can see exactly what changes I'm making to the get-group-members API (adding the `roles`)